impl-endpoints/boringssl/run_endpoint.sh

```diff
@@ -14,5 +14,6 @@ else
     echo "Running BoringSSL server."
     echo "Server params: $SERVER_PARAMS"
     echo "Test case: $TESTCASE"
-    bssl server -accept 4433 -cert /test-inputs/example.crt -key /test-inputs/example.key -subcert /test-inputs/dc.txt
+    DC=$(find /test-inputs/ -type f -name \*.txt | shuf -n 1)
```

Comment:
I would rather there be no selection of this sort in the endpoint, since it would have to be repeated everywhere, and, like you noted, it doesn't easily apply to certs.

What I was thinking was having a `make-root -alg 0` or `make-root -alg 0xFFFF` option (similarly for `make-intermediate` and `make-dc`), that would prompt the corresponding functions to randomly pick a signature algorithm from the list [here](https://github.com/xvzcf/tls-interop-runner/blob/dbb655f4a34cdab4ddfbf70b6611c78de593fb80/cmd/util/common.go#L34) and generate the artifact. The endpoints can then pick these up as usual.

Reply:
Right. But the point was having somewhere all of the generated DC as tests vectors, which means generating them all at the same time, rather than making them be randomly generated each one every time,

Reply:
We talk this offline, and have an strategy going forward.

Reply:
Ok. I added a random way to choose in the Makefile itself. But this will be replaced later for golang generation itself.